Packages/DV3D/StructuredGridPlot.py

```diff
@@ -396,11 +396,11 @@ def buildBaseMap(self):
         self.mapManager = MapManager( roi = self.roi )
         self.renderer.AddActor( self.mapManager.getBaseMapActor() )
 
-#         if self.baseMapActor <> None: self.renderer.RemoveActor( self.baseMapActor )
-#         world_map =  None
-#         map_border_size = 20
-#
-#         self.y0 = -90.0
+#         if self.baseMapActor <> None: self.renderer.RemoveActor( self.baseMapActor )               
```

Comment:
did you source "setup_for_development.sh" in "Scripts" directory? It shouldn't let you commit with trailing spaces